packages/slate-react/src/components/leaf.tsx

```diff
@@ -53,17 +53,10 @@ const Leaf = (props: {
           ref={placeholderRef}
           contentEditable={false}
           style={{
+            position: 'absolute',
```

Comment:
Can you explain the changes here? Why isn't it structured so that the only thing rendered is the placeholder returned by `renderPlaceholder`? Instead of the wrapping `<span>` that remains here?

Reply:
I wrote it this way thinking it'd be safer for the user but I agree with you this is more consistent with the rest of slate :+1:

Addressed